kdestandardize.ado

```diff
@@ -179,8 +179,21 @@ prog def kdestandardize
 					   6 `"Percentage of Teacher Turnover"'                  ///
 					   7 `"School Leadership Composite"', modify
 
+	   // Define EQ_LABEL
+	   la def eqlabel 1`"Exemplary/Accomplished"'                            ///
+					  2`"High/Expected"'                                     ///
+					  3`"School-Level"'                                      ///
+					  4`"Strongly Agree/Agree"'                              ///
+					  5`" "', modify
```

Comment:
We shouldn't have a value label for an empty string.  What would the value of a space convey - if someone were able to see the value?